apps/web/src/components/NftSelector/NftSelector.tsx

```diff
@@ -215,12 +223,19 @@ function NftSelectorInner({ onSelectToken, headerText, preSelectedContract, even
     return filteredTokens;
   }, [filterType, isSearching, network, sortType, tokenSearchResults]);
 
-  const ownsWalletFromSelectedChainFamily = doesUserOwnWalletFromChainFamily(network, query);
+  const ownsWalletFromSelectedChainFamily = doesUserOwnWalletFromChainFamily(
+    network as Chain,
+    query
+  );
```

Comment:
maybe we should update the underlying `doesUserOwnWalletFromChainFamily` to feature:
```
if (chain === 'All Networks') {
  return true
}
```